test/utils/contracts.ts

```diff
@@ -9,6 +9,7 @@ contract Test {
 
     function expose() public returns (address handler) {
         bytes32 slot = 0x6c9a6c4a39284e37ed1cf53d337577d14212a4870fb976a4366c693b939918d5;
+        /// @solidity memory-safe-assembly
```

Comment:
Very complete - but this one is probably not really needed (since it isn't compiled with the project anyway).

Reply:
Done 👍🏾